### DISCUSSION TITLE
How to resize the buttons in the whiskermenu

### DISCUSSION THREAD

**Discussion by robson-66** [lah7/gtk3-classic#60] [Help & Support]
In earlier versions of `gtk3-classic` the buttons looked like this 

![ScreenShot](https://i.imgur.com/ZuZznwa.png)

Now they look like this

![ScreenShot](https://i.imgur.com/QZ5SHRS.png)

How to restore the original appearance?

**Comment by lah7** [comment]
I haven't used `whiskermenu`, so I don't know exactly. There are many things that could be the cause:

* Does it look any different if you install normal `gtk3`? 
* Could `whiskermenu`'s code have changed?
* Could a theme have caused it?
* Could something have changed in GTK3 upstream?

Assuming you're using Arch, there are old versions of [`gtk3-classic` in the release notes](https://github.com/lah7/gtk3-classic/releases) you can try, and old [`gtk3`](https://archive.archlinux.org/packages/g/gtk3/) packages in the archive to cross reference to know if it was the patches or general GTK3 version that caused it.

Be sure to reload `whiskermenu` after any package changes so the correct build of GTK3 is loaded.

**Comment by robson-66** [comment]
After installing `gtk3` there is no such problem.
And `whiskermenu` I use the old version built by myself, because the new one works badly with `compiz`.

I use Arch, by the way.

Edit. 
In version `gtk3-classic` 3.24.28 this problem does not occur. 
Maybe you could tell me what patches you added after this version.

**Comment by robson-66** [comment]
I found the source causing this problem, it is `other__default-settings.patch` and after removing it and rebuilding `gtk3-classic` `whiskermenu` it looks like before.

**Reply by lah7** [reply]
Glad it's working again. I'm really not sure why it would be that one specifically - since that patch has hardly changed since it was first added and doesn't directly concern icons or spacing - unless it has something to do with overlay scrollbars.

**Comment by robson-66** [comment]
Well, this problem appeared in version 3.24.29 after adding this patch. At first I thought that this change occurred in `gtk3` but after installing `gtk3` this problem did not occur.
